tests/test_transcription.py

```diff
@@ -64,3 +65,14 @@ def test_transcribe_prerecorded_find_and_replace_list():
     )
     actual_transcript = response["results"]["channels"][0]["alternatives"][0]["transcript"]
     assert actual_transcript == MOCK_TRANSCRIPT.replace("fast", "slow").replace("miss", "snooze")
+
+
+@pytest.mark.asyncio
+async def test_transcribe_prerecorded_file():
+    """
+    Test basic asyncronous pre-recorded transcription.
+    """
+    example_wav_file = os.path.join(CURRENT_DIRECTORY, "the-missle-short.wav")
```

Comment:
"missile"

Reply:
The file "the-missle-short.wav" does not exist

Reply:
Fixed, thanks!